jenkins/scripts/cbts/coverage_utils/pystart_report.py

```
@@ -0,0 +1,307 @@
+                continue
+            full = os.path.join(dirpath, nm)
+            rel = canon(os.path.abspath(full))
+            try:
+                tree = ast.parse(open(full, encoding="utf-8").read())
+            except (OSError, SyntaxError):
+                continue
+
+            def visit(node, prefix):
+                for child in node.body:
+                    if isinstance(child, (ast.FunctionDef, ast.AsyncFunctionDef)):
+                        funcs.add((rel, prefix + child.name))
```

Ruff (0.15.20)

[warning] Function definition does not bind loop variable `rel`

(B023)